master/buildbot/steps/package/rpm/rpmbuild.py

```diff
@@ -88,6 +94,9 @@ def __init__(self,
             'stdio', logobserver.LineConsumerLogObserver(self.logConsumer))
 
     def start(self):
+
+        self.rpmbuild = ('%s --define "dist %s"' % (self.rpmbuild, self.dist))
```

Comment:
It would be nicer to create a new local variable for this, rather than modifying in place.  Step instances aren't re-used, but absent that knowledge this looks like every run of the step would add an additional `--define dist` argument.